chia/full_node/full_node.py

```diff
@@ -1111,27 +1112,75 @@ async def sync_from_fork_point(
         blockchain = AugmentedBlockchain(self.blockchain)
 
         async def fetch_blocks(output_queue: asyncio.Queue[Optional[tuple[WSChiaConnection, list[FullBlock]]]]) -> None:
+            # the rate limit for respond_blocks is 100 messages / 60 seconds.
+            # But the limit is scaled to 30% for outbound messages, so that's 30
+            # messages per 60 seconds.
+            # That's 2 seconds per request.
+            seconds_per_request = 2
```

Comment:
Do you want to factor in that the percentage is configurable in config.yaml as `outbound_rate_limit_percent`

Reply:
well, this code assumes that the user doesn't change it. I don't think we can do better than that. There's no message that negotiate what rate limits are in effect.

Reply:
I see - well that item is very rarely changed so it's likely an ok assumption